### PR TITLE
 refactor(frontend): normalize margins

### PR DESCRIPTION
**Pull request by UnbearableBear**
Approved by aurélie and virginie already

before:
Screenshot 2019-07-04 at 18 59 25
Screenshot 2019-07-04 at 18 59 39

after: 
Screenshot 2019-07-04 at 18 58 54
Screenshot 2019-07-04 at 18 59 10

### CODE REVIEW

File: packages/code-du-travail-frontend/src/outils/indemniteLicenciement/StepItems.js [hunk 1]
```diff
@@ -23,33 +23,34 @@ export function StepItems({ activeIndex = 0, items = [] }) {
...
   display: flex;
   flex-wrap: wrap;
 `;
 
-const StepBase = styled.span`
-  text-align: left;
-  flex-basis: 130px;
+const StepWrapper = styled.span`
   display: flex;
+  width: 135px;
```
Comment (lionelB) [inline packages/code-du-travail-frontend/src/outils/indemniteLicenciement/StepItems.js]: in `em` / `rem` please :)

**Review by lionelB — CHANGES_REQUESTED** [review]
Nice !  Can you document a little bit about wrapper large usage and perhaps general rules used for spacing ?

File: packages/code-du-travail-frontend/src/outils/indemniteLicenciement/StepItems.js (hunk 1 above)
Comment (UnbearableBear) [reply packages/code-du-travail-frontend/src/outils/indemniteLicenciement/StepItems.js]: 👌

**Comment by UnbearableBear** [comment]
Yes I'm going to add the explanations in the UI

**Review by lionelB — APPROVED** [review]
👍

**Pull request merged**